src/app/helps/[id]/page.jsx

```diff
@@ -41,6 +42,16 @@ async function getHelp(helpId) {
   }
 }
 
+
+
+const urlToShare = 'https://example.com'; // Replace with your URL
```

Comment:
we already have the shareUrl inside the component

Reply:
i did't notice it's now a paramter